src/components/Dashboard/TodoList.vue

```diff
@@ -1019,9 +1019,6 @@ export default class TodoList extends Mixins(AllowableActionsMixin, DateMixin, E
 
     if (header) {
       switch (header.name) {
-        case FilingTypes.ADMIN_FREEZE:
-          // do nothing for admin_freeze
-          break
```

Comment:
Admin Freeze/Unfreeze filings will never appear in the Todo List because they cannot be saved as a draft.